api/docker-compose.deploy.yml

```diff
@@ -2,7 +2,7 @@ version: "3"
 
 networks:
   traefik_default:
-    external: true
+    external: false
```

Comment:
why this change?

Reply:
The current redeploy didn't work because it requires a network being created beforehand.  This is just to create the network, rather than expecting an existing network.

I could also just put the network creation command in CLI somewhere, do you think this is throwing of HTTPS?

Reply:
I think that's fine as is actually